docker/deploy-fedimintd/docker-compose.yaml

```diff
@@ -29,7 +29,7 @@ services:
       - "8333:8333"
     command:
       -printtoconsole
-      -rpcallowip=172.17.0.0/16
+      -rpcallowip=0.0.0.0/0
```

Comment:
Getting `403 Forbidden` without these changes. Docker assigned IPs outside of `172.17...`

```
root@ubuntu-4gb-fsn1-2:~/fedimint-docker# docker-compose exec fedimintd bash
bash-5.2# ip addr show eth0
374: eth0@if375: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1500 qdisc noqueue
    link/ether 02:42:c0:a8:20:03 brd ff:ff:ff:ff:ff:ff
    inet 192.168.32.3/20 brd 192.168.47.255 scope global eth0
```

We can take another approach if we're concerned about allowing requests from all IPs, but since it's a staging env I don't think it's necessary.

Reply:
```
version: '3.8'

services:
  app:
    image: my-app:latest
    networks:
      my_network:
        ipv4_address: 172.20.0.10  # Optional: Assign a specific IP to this service

  db:
    image: postgres:latest
    networks:
      my_network:
        ipv4_address: 172.20.0.11  # Optional: Assign a specific IP to this service

networks:
  my_network:
    driver: bridge
    ipam:
      config:
        - subnet: 172.20.0.0/16  # Define the subnet you want to use
          gateway: 172.20.0.1    # Optional: Define the gateway
```

AI tells me we could force the network we want. @bradleystachurski

Reply:
@bradleystachurski https://github.com/fedimint/fedimint/pull/6913